### PR TITLE
global: update dependencies

### PR DESCRIPTION
**Pull request by NRodriguezcuellar**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -59,6 +56,7 @@
     "WTForms<3.0.0,>=2.3.3",
     # invenio-celery conflict fix
     "celery<5.2,>=5.1.0",
+    "flask>=2.0.0"
```
Comment (kprzerwa) [inline setup.py:59]: instead of this you could bump invenio-base, same as we have done in invenio-app-ils

Comment (NRodriguezcuellar) [reply setup.py:59]: ah ofc, i was wondering why it was not setting flask 2 automatically :sweat_smile:

**Review by kprzerwa — APPROVED** [review]

**Review by Pineirin — APPROVED** [review]
LGTM.

File: setup.py [hunk 2]
```diff
@@ -37,19 +37,16 @@
     "uwsgi-tools>=1.1.1",
     "fuzzywuzzy>=0.18.0",
     "python-ldap>=3.2.0,<3.3.0",
-    "invenio-app-ils[lorem,elasticsearch7,postgresql]==1.0.0a56",
+    "invenio-app-ils[lorem,elasticsearch7,postgresql]==1.0.0a57",
     "invenio-assets>=1.2.6",
-    "invenio-base>=1.2.4",
+    "invenio-base>=1.2.5",
```
Comment (zzacharo) [inline setup.py:42]: Can you please remove these deps?

File: setup.py [hunk 3]
```diff
@@ -37,19 +37,16 @@
     "uwsgi-tools>=1.1.1",
     "fuzzywuzzy>=0.18.0",
     "python-ldap>=3.2.0,<3.3.0",
-    "invenio-app-ils[lorem,elasticsearch7,postgresql]==1.0.0a56",
+    "invenio-app-ils[lorem,elasticsearch7,postgresql]==1.0.0a57",
     "invenio-assets>=1.2.6",
```
Comment (zzacharo) [inline setup.py:41]: Same here

**Review by zzacharo — APPROVED** [review]

**Review by ntarocco — APPROVED** [review]

**Pull request merged**